Gacha log documentation (#43)
studiobutter/hoyo-buddy-wiki: docs/gacha-log.md

# Gacha Log URL

Gacha URL is a very long link containing your Gacha Pull History (i.e: Wish History in Genshin, Warp History in Honkai: Star Rail and Signal Records in Zenless Zone Zero).

## Obtaining the URL

### PC Players

> Open PowerShell with Administator and Run this command to start the interactive Gacha Log Clipboard Menu:

Reviewer: Is administrator permission absolutely needed?

Author: Doesn't need administrator, will be removing this part

```powershell

> Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex "&{$((New-Object System.Net.WebClient).DownloadString('https://gacha.studiobutter.io.vn/Copy-Menu.ps1?ref_type=heads'))}"

Reviewer: We would need to localize the scripts as well.

Author: Will tackle this later. [studiobutter/gacha-stuff/issues/#1](https://github.com/studiobutter/gacha-stuff/issues/1)

Author: Script LocalizedLocalized

Author: Updated the script

```

See Source Code [here](https://github.com/studiobutter/gacha-stuff)

> ## iOS Players

Reviewer: ```suggestion
### iOS Players
```

> Follow this video tutorial to see how to obtain the Gacha Log Link

Reviewer: ```suggestion
Follow this video tutorial to get your gacha log URL on iOS.
```

<iframe width="560" height="315" src="https://www.youtube.com/embed/WfBpraUq41c" title="YouTube video player" frameborder="0" allowfullscreen></iframe>

## Android players

> Follow this video tutorial to see how to obtain the Gacha Log Link

Reviewer: ```suggestion
Follow this video tutorial to get your gacha log URL on Android.
```
